src/diem/testing/miniwallet/app/static/openapi.yaml

```diff
@@ -126,19 +121,16 @@ paths:
       responses:
         202:
           description: >
-              Server accepted the sending payment request.
-
-
-              * Server may reflect the result on account balances immediately as the fund is considered as
-                in the process of transferring out and should not be used for other purpose.
-              * A client should expect the payment will be sent in short time after received the response,
-                but it is not a limit to the server to complete the payment before respond the request.
-              * A client can confirm the payment is completed by the following ways:
-                1. Both sender and receiver account balances are updated accordingly.
-                2. Sender or receiver application exposes an event by [get account events endpoint](#get-/accounts/-account_id-/events).
-              * There is no clear way to confirm the action is failed unless server exposes an event by
-                [get account events endpoint](#get-/accounts/-account_id-/events).
-
+              The server accepts the payment request. 
+              * The server may reflect the payment sent in the account balance immediately. 
+              The funds being sent are considered to be in the process of being transferred 
+              out and should not be used for any other purpose. 
+              * Once you receive the response, expect the payment to be sent in a short time. 
+              But it is not a limit to the server to complete the payment before respond the request.
+              * You can confirm if the payment is completed by:
+                    * Checking if the sending and receiving account balances have been updated accordingly. 
+                    * Checking if the sending or receiving wallet app exposes an event using the get account events endpoint. 
+              * Use the get account events endpoint to confirm if the action has failed. 
```

Comment:
`get account events endpoint` is optional to implement, thus a server may not implement it and no account events change be checked.